### DISCUSSION TITLE
Scriptable context to align to CHART.JS

### DISCUSSION THREAD

**Discussion by stockiNail** [chartjs/chartjs-plugin-zoom#499] [Q&A]
Having a look to `mode` and `overScaleMode`, I see they are scriptable but the created context passed to callback seems:

1. missing of type attribute
2. is not persistent therefore every invocation creates new object

I was thinking to propose a PR to add the scriptable column to the options but I'm not sure if this is something that we want, because also the defaults are defined without the `descriptors`.

**Comment by kurkle** [comment]
It is something I've thought of, but have been postponing until critical bugs are fixed
